iOS/WAStickersThirdParty/WebPManager.swift

```diff
@@ -18,6 +18,33 @@ class WebPManager {
         return decoder.frameCount > 1
     }
 
+    func minFrameDuration(webPData data: Data) -> TimeInterval {
+        guard let decoder = YYImageDecoder(data: data, scale: 1.0) else { return -1 }
+        guard decoder.frameCount > 1 else { return -1 }
+
+        var minFrameDuration = decoder.frameDuration(at: 0)
+        for index in 1..<decoder.frameCount {
```

Comment:
The frame count doesn't start from 0?

Reply:
We initialize `minFrameDuration` with the duration at 0!